guides/integrate-firebase.md

```diff
@@ -0,0 +1,272 @@
+
+This guide will cover:
+
+- How to configure Beyond Identity as an Identity Provider
+- How to create an identity and generate a passkey
+- How to authenticate with a passkey
+- How to configure the OpenID Connect plugin
+
+## Prerequisites
+
+- Set up a [developer account](/docs/v1/workflows/account-setup)
+
+## Firebase
+
+- [Authenticate Using OpenID Connect on Android](https://firebase.google.com/docs/auth/android/openid-connect)
+- [Authenticate Using OpenID Connect on Apple platforms](https://firebase.google.com/docs/auth/ios/openid-connect)
+
+## Install the Embedded SDK
+
+In order to use Beyond Identity functionality in your application, you will need to install the SDK. The [Android SDK](https://github.com/gobeyondidentity/bi-sdk-android) and the [Swift SDK](https://github.com/gobeyondidentity/bi-sdk-swift) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
+
+## Initialize the Embedded SDK
+
+Once you've installed the SDK, initialize it so that you can call the Embedded functions.
+
+<Tabs groupId="sdks" queryString>
+<TabItem value="kotlin" label="Kotlin">
+
+<SetupKotlin />
+
+</TabItem>
+<TabItem value="swift" label="Swift">
+
+<SetupSwift />
+
+</TabItem>
+</Tabs>
+
+## Set up Beyond Identity as an Identity Provider
+
+To set up Beyond Identity as an Identity Provider, you need to create a [Realm](/docs/v1/workflows/realms) to hold identities and configuration. Inside that realm, you'll need to create an [Application](/docs/v1/workflows/applications) that contains the authentication flow configuration. These can be configured in you admin console that was created for you when you signed up for a developer account.
+
+### Create a Realm
+
+From the admin console, click the realm drop-down, and click **Create new realm**.
+
+<Arcade clip={Clip.CreateRealm} />
+
+### Create an Application
+
+From the admin console, click **Apps**, then click **Add app**.
+
+There is a lot to configure when creating an application. When creating your application make sure:
+
+- Redirect URIs contains your application's App Scheme or Universal URL
+- Configuration Type (in the Authenticator Config tab) is set to `Embedded SDK`
+- Invoke URL (in the Authenticator Config tab) contains your application's App Scheme or Universal URL
+- Invocation Type (in the Authenticator Config tab) is set to `Automatic`.
+
+For help choosing options, visit the following guides:
+
+- [Workflows: Applications](/docs/v1/workflows/applications)
+- [Authenticator Config](/docs/v1/platform-overview/authenticator-config)
+
+<Arcade clip={Clip.CreateApplication} />
+
+Now that we have created our application, we are ready to configure our custom OpenID Connect provider in the [Firebase console](https://console.firebase.google.com).
+
+![Firebase Step 1](/assets/firebase-oidc-1.png)
```

Comment:
It might be nice for this First image to have fake default values.

Name could have "Beyond Identity"
Client ID could have "Your New Application -> External Protocol -> Client ID"

Reply:
Yeah, good call